packages/opentelemetry-node/test/instr-aws-sdk.test.js

```diff
@@ -41,7 +41,7 @@ const testFixtures = [
             //          span b592a3 "manual-parent-span" (26.1ms, SPAN_KIND_INTERNAL)
             //     +4ms `- span bbe07e "S3.ListBuckets" (21.5ms, SPAN_KIND_CLIENT)
             //    +10ms   `- span b3b885 "GET" (7.0ms, SPAN_KIND_CLIENT, GET http://localhost:4566/?x-id=ListBuckets -> 200)
-            const spans = col.sortedSpans;
+            const spans = filterOutGcpDetectorSpans(col.sortedSpans);
```

Comment:
What's changed? Is this happening on main as well, do you know?

Reply:
Answer: With just the new HTTP semantics, this no longer works: https://github.com/elastic/elastic-otel-node/blob/33a92efaae9bc7772ed1a83fdab91f165923fd84/packages/opentelemetry-node/test/testutils.js#L403